packages/console/src/App.tsx

```diff
@@ -12,183 +11,62 @@ import './scss/overlayscrollbars.scss';
-                </Route>
-                <Route path="sign-in-experience">
-                  <Route
-                    index
-                    element={<Navigate replace to={SignInExperiencePage.BrandingTab} />}
-                  />
-                  <Route path=":tab" element={<SignInExperience />} />
-                </Route>
-                <Route path="connectors">
-                  <Route index element={<Navigate replace to={ConnectorsTabs.Passwordless} />} />
-                  <Route path=":tab" element={<Connectors />} />
-                  <Route path=":tab/create/:createType" element={<Connectors />} />
-                  <Route path=":tab/:connectorId" element={<ConnectorDetails />} />
-                </Route>
-                <Route path="users">
-                  <Route index element={<Users />} />
-                  <Route path="create" element={<Users />} />
-                  <Route path=":id" element={<UserDetails />}>
-                    <Route index element={<Navigate replace to={UserDetailsTabs.Settings} />} />
-                    <Route path={UserDetailsTabs.Settings} element={<UserSettings />} />
-                    <Route path={UserDetailsTabs.Roles} element={<UserRoles />} />
-                    <Route path={UserDetailsTabs.Logs} element={<UserLogs />} />
-                  </Route>
-                  <Route
-                    path={`:id/${UserDetailsTabs.Logs}/:logId`}
-                    element={<AuditLogDetails />}
-                  />
-                </Route>
-                <Route path="audit-logs">
-                  <Route index element={<AuditLogs />} />
-                  <Route path=":logId" element={<AuditLogDetails />} />
-                </Route>
-                <Route path="roles">
-                  <Route index element={<Roles />} />
-                  <Route path="create" element={<Roles />} />
-                  <Route path=":id" element={<RoleDetails />}>
-                    <Route index element={<Navigate replace to={RoleDetailsTabs.Settings} />} />
-                    <Route path={RoleDetailsTabs.Settings} element={<RoleSettings />} />
-                    <Route path={RoleDetailsTabs.Permissions} element={<RolePermissions />} />
-                    <Route path={RoleDetailsTabs.Users} element={<RoleUsers />} />
-                  </Route>
-                </Route>
-                <Route path="settings" element={<Settings />} />
-                <Route path="profile" element={<Profile />} />
-              </Route>
-            </Route>
-          </Routes>
-        </AppBoundary>
-      </SWRConfig>
-    </ErrorBoundary>
+    <LogtoProvider
+      config={{
+        endpoint: adminTenantEndpoint,
+        appId: adminConsoleApplicationId,
+        resources,
+        scopes,
+      }}
+    >
+      {!isCloud || (data && isSettle && currentTenantId) ? (
+        <AppEndpointsProvider>
+          <Main />
+        </AppEndpointsProvider>
+      ) : (
+        <CloudApp />
```

Comment:
The `<CloudApp />` will need the `AppEndpoint` since the live preview entry is in it.

Reply:
cloud app doesn't need app endpoint context. it's a onboarding process which has a specific tenant id to use

Reply:
i would prefer to create the actual tenant AFTER onboarding - reduce unused data